packages/gatsby/src/utils/webpack.config.js

```diff
@@ -332,6 +333,11 @@ module.exports = async (
         ),
         "create-react-context": directoryPath(`.cache/create-react-context.js`),
       },
+      plugins: [
+        PnpWebpackPlugin.bind(directoryPath(`.cache`), module),
+        PnpWebpackPlugin.bind(directoryPath(`public`), module),
```

Comment:
Is this necessary for `public`?

Reply:
iirc some files in `public` depend on `gatsby-link`, which is a `gatsby` dependency rather than something listed in the user project's package.json.